client/src/components/KeyboardShortcutDialog.vue

```diff
@@ -21,12 +21,11 @@ export default defineComponent({
           ['z', 'x'],
         ],
       ],
-      ['Decrease/increase window', [['-', '=']]],
+      ['Decrease/increase window width', [['-', '=']]],
       ['Decrease/increase window level', [['[', ']']]],
       ['Toggle fullscreen', [['e', 'd', 'c']]],
       ['Mark as usable/usable extra/questionable/unusable', [['u', 'i', 'o', 'p']]],
       ['Place crosshairs at location', [['click']]],
-      ['Changing window width & level', [['click + dragging']]],
```

Comment:
We're removing this control?

Reply:
I think we just need to bind it manually. I assume there's some event binding we can listen to from the VTK side; we may have to ask vtk.js devs.

Reply:
I used @floryst's advice to make the changes in [0422c1a](https://github.com/OpenImaging/miqa/pull/478/commits/0422c1a5d2371be5cfd3960dd170ab9b74820b8b), wherein I put the functionality (and keyboard shortcuts row) back in.